Artifacts/MsSql/Data/Security/2040-UpdateParentResourceClaimId.sql

```diff
@@ -0,0 +1,8 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+UPDATE dbo.ResourceClaims 
+SET ParentResourceClaimId = NULL 
+WHERE ResourceName ='people'
```

Comment:
Please use the `ClaimName` column in the `WHERE` clause because it contains the full URI to definitively identify the resource claim, preventing accidental updates of other claims that may exist with the same `ResourceName` of "People".

```sql
WHERE ClaimName = 'http://ed-fi.org/ods/identity/claims/domains/people'
```